### ISSUE TITLE
Mobile: search-bar `search tips and examples` dropdown broken

### ISSUE THREAD

**Issue by dblommesteijn** [EUDAT-B2SHARE/b2share#421]
On mobile devices the search-bar's `search tips and examples` drop-down menu breaks in input alignment of the text-field. Buttons: `search tips` and `search guide` are blocked by the tooltip (disable it on-click).

Original issue: #297

![screen shot 2014-09-22 at 17 56 53](https://cloud.githubusercontent.com/assets/395001/4359352/26ffe7e4-4271-11e4-8347-b03b4e0fd54b.png)

**Comment by emanueldima** [comment]
won't fix in v1, N/A in v2
